server/handlers/requestHandlers.go

```diff
@@ -45,5 +45,23 @@ func (h *Handlers) SaveNewUser(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
+	
+	r.ParseForm()
+	login := &middleware.LoginRequest{
+		Email: r.PostForm.Get("Email"),
+		Password: r.PostForm.Get("Password"),
+	}
+
+	if found := h.db.FindStudent(*login); found == false {
+		log.Println("INFO [handlers/requestHandlers.go] Failed Log In")
+		http.Redirect(w, r, "/", http.StatusSeeOther)
```

Comment:
use http.StatusUnauthorized

Reply:
this is http error 401 and it won't let you redirect for now, but it's the "right" http status to display for a failed login request. instead of this line just do "w.WriteHeader(http.StatusUnauthorized)"